packages/mercury/src/base/_fields.scss

```diff
@@ -36,9 +36,15 @@
     grid-template-columns: max-content 1fr;
 
     &__label {
-      // Make the label vertically aligned with an inline control.
-      // Also do not look center on taller contro, like a textarea.
-      margin-block-start: 11px;
+      // Make the label vertically aligned with an inline control, wihout using
+      // align:center since that property/value will make the label be centered
+      // in a textarea (It's not what's expected.)
+      $labelBlockSize: calc(var(--font-size-body-m) * var(--line-height-tight));
+      transform: translateY(calc(50% - ($labelBlockSize / 2)));
```

Comment:
This won't work for the following reasons:
 - `translateY` is not "friendly" with multiple direction languages. In this case, setting `writing-mode: vertical-lr;`

 - When using textareas (`ch-edit` with multiline), the label is aligned at the middle of the control:
   
   ![image](https://github.com/user-attachments/assets/831d0678-4818-4d98-b931-f4cc106e1f3a)

Try to use `margin-block-start` to properly reserve a space for improving the vertical alignment of the label. If that doesn't work, move the change out of this PR to unlock the "form" classes in the showcase.

Reply:
It also provokes overflow if the label is used at the bottom of an scrollable container

![image](https://github.com/user-attachments/assets/27a0c8bc-bbbf-4b7b-ae9a-b6c0a5cbfd7a)

Reply:
Using `padding-block-start: 8px`:

![image](https://github.com/user-attachments/assets/1e55e22f-4662-4a4f-9a44-cd2f86b427b5)